handler/src/main/java/io/netty/handler/ssl/SslUtils.java

```diff
@@ -349,6 +350,19 @@ static ByteBuf toBase64(ByteBufAllocator allocator, ByteBuf src) {
         return dst;
     }
 
+    /**
+     * Validate that the given hostname can be used in SNI extension.
+     *
+     * @see sun.security.ssl.Utilities#rawToSNIHostName(String)
```

Comment:
@jianglai please remove this as it points to some "private" class that may not be present depending on the JDK you use.